src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -16,13 +16,52 @@
 type: command
 short-summary: Create an App Configuration.
 examples:
-  - name: Create an App Configuration with name, location and resource group.
-    text: az appconfig create -g MyResourceGroup -n MyAppConfiguration -l westus
+  - name: Create an App Configuration with name, location, sku and resource group.
+    text: az appconfig create -g MyResourceGroup -n MyAppConfiguration -l westus --sku Standard
+  - name: Create an App Configuration with name, location, sku and resource group with system assigned identity.
+    text: az appconfig create -g MyResourceGroup -n MyAppConfiguration -l westus --sku Standard --assign-identity [system]
+"""
+
+helps['appconfig identity'] = """
+type: group
+short-summary: Managed identities for App Configurations.
+"""
+
+helps['appconfig identity assign'] = """
+type: command
+short-summary: Update managed identities for an App Configuration.
+examples:
+  - name: Enable the system-assigned identity for an existing an App Configuration
```

Comment:
>an [](start = 62, length = 2)

nit: " ....an existing App Configuration"

Reply:
sure thing

---
In reply to: [364512736](https://github.com/Azure/azure-cli/pull/11795#discussion_r364512736) [](ancestors = 364512736)